KIPs/kip-160.md

```diff
@@ -0,0 +1,74 @@
+---
+kip: 160
+title: An Update of Treasury Fund Rebalancing
+requires: 103
+---
+
+## Simple Summary
+An update of treasury fund rebalancing.
+
+## Abstract
+The treasury fund rebalancing is updated, resulting in updates to the treasury rebalance contract v2 and the related core logic.
+
+## Motivation
+According to kip103, the treasury rebalancing refers to the act of burning existing fund balances and minting new funds. This event happens at a reserved time, such as a hard fork block number. Through the TreasuryRebalance contract, the disclosure of the treasury fund activities can be conducted transparently and verifiably. However, there are a few shortcomings of KIP103; (1) it only permits the decrease in the total balance of the funds, (2) its rebalance blocknumber is immutable.
```

Comment:
How about capitalizing all `kip` to `KIP`?

Reply:
Thanks, revised as adviced. https://github.com/klaytn/kips/pull/160/commits/a1c8190fe19855d01ead450622d849894bf470c3